airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.tsx

```diff
@@ -254,24 +254,35 @@ const useInitialValues = (
   }, [initialSchema, connection, isEditMode, destDefinition]);
 };
 
-const useFrequencyDropdownData = (): DropDownRow.IDataItem[] => {
+const useFrequencyDropdownData = (
+  additionalFrequency: WebBackendConnectionRead["schedule"]
+): DropDownRow.IDataItem[] => {
   const { formatMessage } = useIntl();
 
-  return useMemo(
-    () =>
-      FrequencyConfig.map((item) => ({
-        value: item.config,
-        label: item.config
-          ? formatMessage(
-              {
-                id: `form.every.${item.config.timeUnit}`,
-              },
-              { value: item.config.units }
-            )
-          : formatMessage({ id: "frequency.manual" }),
-      })),
-    [formatMessage]
-  );
+  return useMemo(() => {
+    const frequencies = FrequencyConfig.map((frequencyConfig) => frequencyConfig.config);
+    if (additionalFrequency) {
+      const additionalFreqAlreadyPresent = frequencies.some(
+        (frequency) =>
+          frequency?.timeUnit === additionalFrequency.timeUnit && frequency?.units === additionalFrequency.units
+      );
+      if (!additionalFreqAlreadyPresent) {
+        frequencies.unshift(additionalFrequency);
```

Comment:
I'd actually suggest we add this to the end. I feel it's a bit weird, that this sits above the "Manual" option eventually. Also given that if they have a custom frequency, it might not be a "recommended" once, I feel putting it at the end might be preferred.

Reply:
I have updated this to push it onto the end instead. Here's what it looks like now in the UI:
<img width="985" alt="Screen Shot 2022-08-17 at 10 51 27 AM" src="https://user-images.githubusercontent.com/22731524/185208235-1a391d22-a389-4c83-8964-30cdfa6e2bd6.png">